test/integration/api/integrations_controller_test.rb

```diff
@@ -221,11 +221,16 @@ def test_update_proxy_rule_position
     proxy_attributes = attrs['proxy']
     proxy_attributes.delete('proxy_rules_attributes')
     api_backend = proxy_attributes.delete 'api_backend'
+    policies_config = proxy_attributes.delete 'policies_config'
     proxy.reload
 
     assert_equal proxy.attributes.slice(*proxy_attributes.keys), proxy_attributes
     assert_equal 'http://bye-world-api.3scale.net:80', proxy.api_backend
 
+    assert JSON.parse(policies_config).all? { |policy|
+      proxy.policies_config.detect Proxy::PolicyConfig.new(policy)
+    }
```

Comment:
I just fixed the existing test because direct comparison is not possible. In the same way like it is done for `proxy.api_backend`. I updated the comparison to be simpler.